levels/05_theCorridor.jsx

```diff
@@ -0,0 +1,91 @@
+#BEGIN_PROPERTIES#
+{
+    "version": "0.1",
+    "commandsIntroduced": []
+}
+#END_PROPERTIES#
+/***************
+ * theCorridor.js *
+ ***************
+ */
+
+function startLevel(map) {
+#START_OF_START_LEVEL#
+    map.displayChapter('Chapter 4.1\nChapters are supposed to be more than one level long');
```

Comment:
Well. In the "untrusted" they are:
Chapter 1 Breakout: levels 1 to 7
Chapter 2 Raiders of the Lost Algorithm: levels 8 to 14
Chapter 3 Betrayal: levels 15 to 21